docs/robots/ridgeback/maintenance_ridgeback.mdx

```diff
@@ -4,9 +4,117 @@ sidebar_position: 4
 ---
 
 import ComponentIntroductionRidgeback from "/components/introduction_ridgeback.mdx";
+import GettingNewPackages from "/components/maintenance/getting_new_packages.mdx";
+import Support from "/components/support.mdx";
 
 <ComponentIntroductionRidgeback />
 
-:::warning todo—under construction
+## Battery Replacement
+
+If Ridgeback's batteries need replacement, they are accessible by removing the top-plate and the insulator
+cover. Before performing any service or maintenance to the robot, the battery pack must be fully disconnected.
+The batteries weigh approximately 33 kg (73 lbs) each, so use all necessary care and caution when lifting the
+batteries out of the chassis. Please contact Clearpath Robotics regarding replacement batteries.
+
+## Long-term storage
+
+When storing Ridgeback for long periods of time, its important to properly maintain the batteries to fully
+maximize their life. Consider one of the following two procedures when placing Ridgeback in long-term storage:
+
+- Fully charge Ridgeback, turn it off and put it into storage. Once a week, connect power to the charger
+  and allow the charger to top up the battery for an hour or so.
+- Fully charge Ridgeback, turn it off and put it into storage, but leave the charger connected and powered
```

Comment:
The Ridgeback's charger times out. We should remove the second list item, since the charger will not continue to charge indefinitely.

Reply:
I agree. For long-term storage, we've been suggesting that the charger be connected through a wall timer. The timer should be set to turn off for an hour or two ever 24-hour cycle.

Reply:
I modified the second item to add the note about the wall timer.